docs/admin/reference/admin_interface.rst

```diff
@@ -0,0 +1,277 @@
+The Admin Interface
+===================
+
+The *Admin Interface* is an extended version of the *Journalist Interface*, that
+allows you to manage users and configure the appearance and behaviour of your
+instance's web interfaces.
+
+Logging in
+^^^^^^^^^^
+
+To log in to the *Admin Interface*, start the *Admin Workstation* with persistence
+enabled and double-click the *Journalist Interface* icon on the Desktop. Tor Browser
+will start and load the login page for the *Journalist Interface*. Use your username,
+passphrase, and two-factor authentication token to log in.
+
+By default, you will be logged in to the *Journalist Interface*'s source list page.
+
+|SecureDrop main page|
+
+In the course of normal administration operations you should not need to view source
+communications, but if you do, you can find information on managing submissions in
+the :doc:`journalist guide <../../journalist/journalist>`.
+
+.. note::
+  If you have lost your login information or your two-factor authentication is no longer
+  valid, you can create another account with admin privileges via the command line
+  on the *Application Server*. See :ref:`here <Create Admin CLI>` for more information.
+
+
+.. _User Management:
+
+User Management
+^^^^^^^^^^^^^^^
+
+You can use the *Admin Interface* to add and remove users, and to reset their
+credentials if necessary. To open the *Admin Interface*, click **Admin** in the
+upper right corner of the *Journalist Interface*.
+
+.. _Adding Users:
+
+Adding Users
+^^^^^^^^^^^^
+
+After logging in, you can add new user accounts for the journalists at your organization
+who will be checking the system for submissions. Make sure the journalist is
+physically in the same room as you when you do this, as they will have to be present
+to enable two-factor authentication. SecureDrop supports the use of either a
+smartphone authenticator app or a Yubikey for two-factor authentication. If an
+app is to be used, the journalist should install it before proceeding with the
+account setup.
+
+.. include:: ../../includes/otp-app.txt
+
+#. Click **Admin** in the top right corner of the page to load the *Admin Interface*.
+
+   |SecureDrop admin home|
+
+#. Click **Add User** to add a new user.
+
+   |Add a new user|
+
+#. Hand the keyboard over to the journalist so they can create their own username.
+#. Once they’re done entering a username for themselves, have them save their pre-generated Diceware passphrase to their password manager.
+#. If the new account should also have admin privileges, allowing them to add or delete other journalist accounts, select **Is Admin**.
+#. Finally, set up two-factor authentication for the account, following one of the two procedures below for your chosen method.
+
+.. note::
+   The username **deleted** is reserved, as it is used to mark accounts which
+   have been deleted from the system.
+
+
+FreeOTP
+^^^^^^^
```

Comment:
I think the FreeOTP and YubiKey headlines should be a level below "Adding users".